### ISSUE TITLE
Can't update legal entity without DOB

### ISSUE THREAD

**Issue by quentin-** [stripe/stripe-go#308]
It default to 0 and Stripe API returns a validation error.
It also sends empty string for the type, which is unexpected.

```go
params := &stripe.AccountParams{
  LegalEntity: &stripe.LegalEntity{
    PersonalID: "000-00-0000",
  },
}

account.Update(req.AccountID, params)
```

```
&{values:[
  {Key:legal_entity[type] Value:} 
  {Key:legal_entity[dob][day] Value:0} 
  {Key:legal_entity[dob][month] Value:0} 
  {Key:legal_entity[dob][year] Value:0} 
  {Key:legal_entity[personal_id_number] Value:000-00-0000} 
]}
```

```json
{"field_name":"legal_entity[dob][day]", "code":"", "messages":["Not a valid day"]}
```

**Comment by aravind88** [comment]
@brandur @remistr What would be best way around this?

**Comment by brandur** [comment]
@quentin- @aravind88 A fix has been added in #310 by @guillaumehenriot. We should have it in very soon.
